dashboard/test/controllers/rubrics_controller_test.rb

```diff
@@ -39,4 +39,37 @@ class RubricsControllerTest < ActionController::TestCase
     assert_equal @lesson.id, rubric.lesson_id
     assert_equal 2, learning_goals.length
   end
+
+  test 'submits rubric evaluations of a student' do
+    rubric = create :rubric, lesson: @lesson, level: @level
+    learning_goal1 = create :learning_goal, rubric: rubric
+    learning_goal2 = create :learning_goal, rubric: rubric
+    student = create :student
+    teacher = create :teacher
+    create :learning_goal_evaluation, user: student, learning_goal: learning_goal1, teacher: teacher
+    create :learning_goal_evaluation, user: student, learning_goal: learning_goal2, teacher: teacher
```

Comment:
this looks fine to merge, but you might be able to add a trait to the rubric factory to automatically give it learning goals and evaluations: 
```suggestion
    student = create :student
    teacher = create :teacher
    rubric = create :rubric, lesson: @lesson, level: @level, student: student, teacher: teacher
```
here is the closest example i can find: https://github.com/code-dot-org/code-dot-org/blob/043add79d73dbb22d988edf40b307bca7ff9b9fc/dashboard/test/factories/factories.rb#L892-L900

you should be able to pass the teacher and student down via `evaluator.teacher`, kind of like `evaluator.lessons_count` in the code snippet above. here is an example of a value for lessons_count being passed in, which makes me think you can do the same thing with teacher: https://github.com/code-dot-org/code-dot-org/blob/b74d462639d04d8e1f355a79c361ca13b890b4d3/dashboard/test/helpers/script_levels_helper_test.rb#L11

Reply:
Done!